src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt

```diff
@@ -103,16 +103,20 @@ class ModuleExecutor(
                 }, {
                     log.info("[RESPONSE] [$issue] [${response.first}] Successful")
                 })
+                issueUpdateContextCache.storage.forEach {
+                    it.value.triggeredBy = issue
+                }
             }
 
         issueUpdateContextCache.storage
-            .mapValues { applyIssueChanges(it.value) }
-            .filterValues { it.isLeft() }
-            .forEach { entry ->
-                (entry.value as Either.Left).a.exceptions.forEach {
-                    log.error("[UPDATE] [${entry.key}] Failed", it)
+            .mapValues { Pair(it.value.triggeredBy, applyIssueChanges(it.value)) }
```

Comment:
Do we have a test for the cache in the moduleexecutor?

Reply:
We have one for adding failed tickets, however that's for the module response section instead of the issue update section. So afaik we don't have tests for that, though I might miss something.

I was trying to add a test for this but wasn't sure how to do it